SwiftSyntaxCLI/Sources/swift-syntax-cli/commands/Test.swift

```diff
@@ -0,0 +1,169 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
+import Foundation
+
+struct Test: ParsableCommand, Builder {
+  @OptionGroup
+  var arguments: BuilderArguments
+
+  @Flag(help: "Don't run lit-based tests")
+  var skipLitTests: Bool = false
```

Comment:
fly-by comment, I don't know if you have an existing cli you need to match, but imo these sort of options read better as:
```
@Flag(help: "Run lit-based tests", inversion: .enableDisable)
var litTests: Bool = true
```

Reply:
We’d like to maintain the behavior of `build-script.py` for now. Also, I’d like to get rid of the lit-based test anyway, so this flag can hopefully be defined away soon anyway.